src/psyclone/tests/test_files/dynamo0p3/infrastructure/reference_element_mod.f90

```diff
@@ -0,0 +1,105 @@
+!-----------------------------------------------------------------------------
+! Copyright (c) 2017,  Met Office, on behalf of HMSO and Queen's Printer
+! For further details please refer to the file LICENCE.original which you
+! should have received as part of this distribution.
+!-----------------------------------------------------------------------------
```

Comment:
Please add a link to the `LICENCE.original`, i.e.
```
! LICENCE.original is available from the Met Office Science Repository Service:
! https://code.metoffice.gov.uk/trac/lfric/browser/LFRic/trunk/LICENCE.original
! -----------------------------------------------------------------------------
```

Reply:
Done.